### DISCUSSION TITLE
I'm using PrimeReact in docusaurus, How can I use markdown-syntax for cell's content when rendering table with DataTable?

### DISCUSSION THREAD

**Discussion by xbpk3t** [primefaces/community#701] [PrimeReact] [answered]
Currently cell's content are in markdown-syntax and will eventually render as a raw string

Hope will preview like this [Components | MDX](https://mdxjs.com/table-of-components/)

**Comment by mertsincan** [comment, marked as answer]
Hi,

DataTable with cell has templatings named `editor` and `body`. I think you can set a 3rd party markdown input and parser it to them.
